openlibrary/templates/lib/header_dropdown.html

```diff
@@ -57,6 +61,9 @@
                   <button $(tracker)>$(link["text"])</button>
                 </form>
               $else:
+                $if is_privileged_user:
+                  $if (link['text'] == 'Pending Requests') and get_counts_mode(mode='open') > 0:
+                    <div class="mr-notifications-list">$(get_counts_mode(mode='open'))</div>
```

Comment:
```suggestion
                    <div class="app-drawer__badge">$(cached_get_counts_by_mode(mode='open'))</div>
```
Looks like there is already a class for app drawer badges.